controllers/users.js

```diff
@@ -622,6 +622,18 @@ const updateRoles = async (req, res) => {
   }
 };
 
+const archiveUserIfNotInDiscord = async (req, res) => {
+  try {
+    const data = await userQuery.archiveUserIfNotInDiscord();
```

Comment:
what if data is no users updated, send proper response.
Response will still send: Successfully updated users archived role to true if in_discord role is false